etcd/server/service.sls

```diff
@@ -9,8 +9,10 @@ etcd_packages:
 {%- if server.get('engine', 'systemd') %}
   - require:
     - file: /etc/default/etcd
+  {% if not grains.get('noservices', False) %}
```

Comment:
That grains is not only for testing but for case when you want to apply states without touching services (starting, restarting, etc.).